std_msgs/Float64 first_x
std_msgs/Float64 first_y
std_msgs/Float64 first_w

std_msgs/Float64 second_x
std_msgs/Float64 second_y
std_msgs/Float64 second_w

std_msgs/Bool   check_pos